### ISSUE TITLE
Doesn't Work MC 1.8.1

### ISSUE THREAD

**Issue by diegobh** [xXKeyleXx/MyPet#428]
I've Spigot 1.8 and download the version: MyPet-1.8-1.2.1-SNAPSHOT.jar	

I've tryed to capture the mob with lead but doesn't show any damage (hp life) and the mobs just die.

I tryied a lot of mobs: skeleton, zumbis, pigs .. no one worked.

How to fix it, please?

Thank you!

**Comment by xXKeyleXx** [comment]
Any errors in the server log?

**Comment by diegobh** [comment]
No. All ok.

ServerLog: http://pastebin.com/NEd8YTgn

MyPet log: http://pastebin.com/ptUALjqu

config.yml: http://pastebin.com/VF3FsP9M

worldgroups.yml: http://pastebin.com/DRB2VQ2D

permissions.yml: http://pastebin.com/UN3y9LNT (PermissionsEx)

Ive tryied atk the mobs with the Lead (LeashItem: 420) but no shows the damage (mob's hp) and the mob died.

Thank you!

**Comment by xXKeyleXx** [comment]
Your permissions are looking good and everything else too but I don't know why it's not working because everything works fine for me :/

**Comment by diegobh** [comment]
Ive tryied to create a new world too, but it is the same.

The version 1.8 really doesn't work on Spigot 1.8. =(

Wich Java's version have u used to compile MyPet?

My server is running with Java 7.
I had some problems with plugins compiled with Java 8, but the consoles told me Java error, that dont show with MyPet.
When i compiled the same source with Java 7, i didn't get any console errors and the plugins worked fine.

**Comment by xXKeyleXx** [comment]
I use Java 6 to compile MyPet because Spigot/Craftbukkit are compiled with Java 6 too.
Could you please upload your Spigot version so I can test it by myself.

**Comment by diegobh** [comment]
Sure!

Im using spigot-1.8.jar : http://goo.gl/zBA0tN

Thank you!!!

**Comment by xXKeyleXx** [comment]
Everything works fine for me (downloaded your spigot and the latest MyPet build). It has to be your configuration, most likely the permissions.

**Comment by diegobh** [comment]
Hum... o/ i found something interesting.

I've edited my permissions.yml and now i can tamme pets.

But...
1- It yet doesn't show me the monster's hp after i hit it with LeashItem...
2- I set the mobskill to Farm, but it isn't atk the mobs and it isn't pickup itens on floor...
3- I can tamme only one pet. If i want a new pet, i have to release that ive already tammed.

But im happy! It was a error with PEX, the plugin is working well. I thing this "bugs" is all miss configs lol

Thank you xXKeyleXx for making this wonderfull plugin and thank you very much for your attention and help :D
